assets/app/view/game/part/label.rb

```diff
@@ -16,25 +16,25 @@ class Label < Base
 
         P_LEFT_CORNER = {
           flat: {
-            region_weights: { LEFT_CORNER => 1.0, (LEFT_CENTER + LEFT_MID) => 0.5 },
+            region_weights: { LEFT_CORNER => 1.0, LEFT_MID => 0.25 },
             x: -71.25,
             y: 0,
           },
           pointy: {
-            region_weights: { LEFT_CORNER => 1.0, (LEFT_CENTER + LEFT_MID) => 0.5 },
+            region_weights: { LEFT_CORNER => 1.0, LEFT_MID => 0.25 },
             x: -67,
             y: 0,
           },
         }.freeze
 
         P_BOTTOM_LEFT_CORNER = {
           flat: {
-            region_weights: { [19, 20] => 1.0, [21] => 0.5 },
+            region_weights: { BOTTOM_LEFT_CORNER => 1.0, [21] => 0.5 },
```

Comment:
i think you're going to conflict with me here

Reply:
Merged with master, no conflicts. I think I merged with your changes